examples/deck/example.js

```diff
@@ -282,12 +282,22 @@ export default class Example extends Component<
       width: window.innerWidth
     };
 
+    let { modeConfig } = this.state;
+    if (mode === 'drawCircleByBoundingBox') {
```

Comment:
This will cause extra re-renders since you're creating a new object each time. Instead do `this.setState({ modeConfig: ... })`. See how `booleanOperation` is implemented.